### DISCUSSION TITLE
Private Endpoints created by integration ADF are stuck in Pending state

### DISCUSSION THREAD

**Discussion by renepajta** [Azure/data-landing-zone#165] [Q&A] [answered]
**Describe the bug**
After 15+ minutes, private endpoints created by ADF are in Pending state and they need to be manually approved. 

The deployment finish successfully, however, the linked services which leverages PEs won't work until PEs are approved. 

**Steps to reproduce**

1.  Deploy Data Landing Zone
2.  Check managed private endpoints for ADF in integration services

**Screenshots**
![image](https://user-images.githubusercontent.com/6043628/127861466-acd08ca2-82b2-40d0-894d-2627abb76b2a.png)
![image](https://user-images.githubusercontent.com/6043628/127861543-1ea48a47-3bc5-4f07-a7c1-e6c909d73f9c.png)
![image](https://user-images.githubusercontent.com/6043628/127861652-f7253e54-c708-4914-bf4d-0a8051ae991f.png)
![image](https://user-images.githubusercontent.com/6043628/127866511-2de97cfc-d94f-482c-9680-5c51f915dbe5.png)

**Comment by marvinbuss** [comment, marked as answer]
@renepajta This is also expected behavior. These private endpoints cannot be created in an approved state. They are created in the MSFT tenant and for security reasons they are always created in a pending state. This has also technical reasons. You can also read about this [here](https://docs.microsoft.com/en-us/azure/data-factory/managed-virtual-network-private-endpoint)
![image](https://user-images.githubusercontent.com/34542414/127867889-0a8a1dfb-1d02-4a2a-9b7f-09213f3eb391.png)
